### ISSUE TITLE
How does the output logits value judge whether it is real or fake?

### ISSUE THREAD

**Issue by Feather06** [ahaliassos/LipForensics#12]
How does the output logits value judge whether it is real or fake?

**Comment by ahaliassos** [comment]
Hi, it is a binary classification task with the label "1" being "fake" and "0" being "real." As a result, the larger the predicted logits, the higher the probability that the model assigns to that sample being fake, and the more negative the logits, the higher the probability that the sample is real. 

Please re-open if it is still not clear.

**Comment by tapas** [comment]
So to predict as real or fake, can I sigmoid(logits) and put some threshold say 0.3 to classify as real or fake ?
